web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/documentElement.js

```diff
@@ -0,0 +1,7 @@
+define( [
+	"./document"
+], function( document ) {
```

Comment:
:no_entry_sign: **Error**: Only one argument is allowed per line in a multi-line function call (*PEAR.Functions.FunctionCallSignature.MultipleArguments*).